packages/components/src/theme/ThemeUtils.test.ts

```diff
@@ -558,9 +558,10 @@ describe.each([undefined, document.createElement('div')])(
         bbb: 'bbb',
       };
 
+      jest.spyOn(window.CSS, 'supports').mockReturnValue(false);
```

Comment:
Test case for when this is true?

Reply:
Should be pretty much all the other test cases. We mocked it to true by default